public/app/features/dashboard/state/DashboardMigrator.ts

```diff
@@ -487,6 +487,18 @@ export class DashboardMigrator {
       });
     }
 
+    if (oldVersion < 22) {
+      panelUpgrades.push((panel: any) => {
+        if (panel.type !== 'table') {
+          return;
+        }
+
+        _.each(panel.styles, style => {
+          style.align = 'auto';
+        });
+      });
+    }
+
```

Comment:
We do not need a migration for this, it's just a new default, and they are applied when the panel is loaded.

Reply:
My bad, I couldn't get existing Table panels to show values in drop down and I thought that was due to migration.

Reply:
Should I remove the migration then?

Reply:
I think since it's a nested default on an array item the default is not applied int the panel constructor like other defaults. 
So think it's a good thing to have this migration, keep it :)

Reply:
Thanks.